source/server/connection_handler_impl.cc

```diff
@@ -26,6 +26,8 @@ void ConnectionHandlerImpl::addListener(Network::ListenerConfig& config) {
   } else {
     ASSERT(socket_type == Network::Address::SocketType::Datagram,
            "Only datagram/stream listener supported");
+    RELEASE_ASSERT(config.udpListenerFactory() != nullptr,
+                   "UDP listner factory is not initialized.");
```

Comment:
Can you add a TODO to make this a proper config check?

Reply:
I think you can remove this assert then? It would crash in an obvious way on the next line.

Reply:
done

Reply:
I think you didn't remove it, just converted it.  Would it actually fail under createActiveUdpListener or is this assert required?

Reply:
ListenerConfig.udpListenerFactory() is populated to default RawUdp listener if not specified in config. So this ASSERT is to check we actually have logic to do this before reaching here. Otherwise below `config.udpListenerFactory()->createActiveUdpListener(*this, dispatcher_, logger_, config);` would crash because of accessing members of nullptr.

Reply:
Gotcha.  If we plan to keep it in, let's at least change
listner -> listener

Reply:
done